scheduler/generic_sched.go

```diff
@@ -452,6 +452,7 @@ func (s *GenericScheduler) computePlacements(place []allocTuple) error {
 				TaskGroup:     missing.TaskGroup.Name,
 				Metrics:       s.ctx.Metrics(),
 				NodeID:        option.Node.ID,
+				Resources:     option.AllocResources,
```

Comment:
You shouldn't have to do anything because the allocation has the task group name and you can look it up through the job